app/experimenter/docs/openapi-schema.json

```diff
@@ -3040,6 +3040,10 @@
             "type": "string",
             "readOnly": true
           },
+          "isRollout": {
+            "type": "string",
```

Comment:
Question: should this be `string` or `boolean`? I was going off the example of `isEnrollmentPaused` right above, which is boolean other places but string here 🤔

Reply:
The Experiment JSONSChema has boolean

https://github.com/mozilla/nimbus-shared/blob/main/types/experiments.ts#L50-L57

Reply:
Yeah that's a bit wonky.  We should look into that separately.